docs/package_integration/uds-package-practices.md

```diff
@@ -1,110 +1,102 @@
 - **Should** minimize network policies to specific selectors needed for Ingress/Egress traffic.
-- **May** template network policy keys to provide flexibility for delivery customers to configure.
-
-### Keycloak
-
-- **Must** use and create a Keycloak client through the `sso` key if the application provides an end-user login.
 - **Should** consider security options during implementation to provide the most secure default possible (i.e. SAML w/SCIM vs OIDC).
 - **Should** name the client `<App> Login` (i.e. `Mattermost Login`) to provide login UX consistency.
 - **Should** clearly mark the client id with the group and app name `uds-<group>-<application>` (i.e. `uds-swf-mattermost`) to provide consistency in the Keycloak UI.
-- **May** end any generated secrets with `-sso` to easily locate them when querying the cluster.
-- **May** template Keycloak fields to provide flexibility for delivery customers to configure.
-
-### Prometheus
-
-- **Must** implement monitors for each application metrics endpoint using it's built-in chart monitors, the `Package` CR `monitor` key, or manual monitors in the config chart.
-
-## Exemptions
-
-UDS Packages **may** make use of the [UDS `Exemption` custom resource](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/operator/README.md#example-uds-exemption-cr) for exempting any Pepr policies, but in doing so they:
-
-- **Must** minimize the scope and number of the exemptions to only what is absolutely required by the application
-- **Must** have documented rationale for any exemptions present
-
-## Structure
+- **Should** limit the use of Zarf variable templates and prioritize configuring packages via Helm value overrides.
+  > This ensures that the package is configured the same way that the bundle would be and avoids any side effect issues of Zarf's `###` templating.
 
-Packages also follow structural guidelines to ensure consistency and flexibility for configuration, they:
+## Silver
 
-- **Must** contain documentation under a `docs` folder at the root that describes how to configure the package and outlines package dependencies.
-  > This allows users of the package to learn more about exposed configuration - it is recommended to make the entrypoint for configuration `configuration.md`.
+_a Silver UDS-Package that integrates with the main features of the UDS Operator, is documented, maintained, and can be confidently operated in production._
 
+- **Must** satisfy all the requirements of [Bronze](#bronze) Packages
+- **Must** define network policies under the `allow` key as required in the [UDS Package Custom Resource](https://github.com/defenseunicorns/uds-core/blob/main/docs/configuration/uds-operator.md)
+- **Must** use and create a Keycloak client through the `sso` key if the application provides an end-user login. [UDS Package Custom Resource](https://github.com/defenseunicorns/uds-core/blob/main/docs/configuration/uds-operator.md)
```

Comment:
```suggestion
- **Must** (except if the application provides no end user login) use and create a Keycloak client through the `sso` key. [UDS Package Custom Resource](https://github.com/defenseunicorns/uds-core/blob/main/docs/configuration/uds-operator.md)
```
Do we want to call out exceptions in a more standard way - there will be apps (especially stuff like DBs) that you wouldn't create a KC client for and it may be good to callout exceptions up front.